.editorconfig

```diff
@@ -0,0 +1,11 @@
+root = true
```

Comment:
@sayak-sarkar @deshmukhmayur do you think we can get rid of this editorconfig and add this editorconfig file in the main mono repo. we don't have to keep sepaate editorconfig files in each microservice

Reply:
ohh sorry please ignore this file is for generator and not microservice